# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/alessandro/vsftpd/fix-detection.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

mail to Chris Evans <chris@scary.beasts.org>

--- ./vsf_findlibs.sh.org	2002-10-28 01:12:53.000000000 +0100
+++ ./vsf_findlibs.sh	2003-04-02 15:01:56.000000000 +0200
@@ -1,7 +1,13 @@
 #!/bin/sh
 # Cheesy hacky location of additional link libraries.
 
-locate_library() { [ ! "$1*" = "`echo $1*`" ]; }
+locate_library() { 
+    while [ "$1" ]; do
+	for i in $1* ; do [ -f "$i" ] && return 0; done
+	shift
+    done
+    return 1
+}
 
 if (grep -q "#define VSF_BUILD_TCPWRAPPERS" builddefs.h); then
   echo "-lwrap";
